metadata.json

```diff
@@ -14,6 +14,7 @@
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
+        "6",
```

Comment:
this is not enough to actually test on CentOS 6
you'll need to add an entry (or two) here
https://github.com/voxpupuli/puppet-bird/blob/eda6e8061adb775309ba429068f266c4ad0be068/.travis.yml

Reply:
fixed :)